app/grandchallenge/evaluation/admin.py

```diff
@@ -35,6 +36,15 @@
     SubmissionGroupObjectPermission,
     SubmissionUserObjectPermission,
 )
+from grandchallenge.evaluation.utils import SubmissionKindChoices
+
+NON_EVALUATION_SOCKET_SLUGS = [
```

Comment:
This seems like duplicate data, shouldn't `NON_ALGORITHM_INTERFACES` be reused?

Reply:
Not sure, it seemed semantically wrong to use `NON_ALGORITHM_INTERFACES` here and I also thought that we might want to distinguish between the two in the future, they just happen to overlap now.

Reply:
Okay then it's probably best to unify them and rename to save them from unintentionally getting out of sync.

Reply:
Ok, makes sense. How about calling the unified list `RESERVED_SOCKET_SLUGS`?